From: Richard B Winters <rik@mmogp.com>
Date: Thu, 23 Apr 2015 16:18:27 -0400
Subject: Fix grammer error in manual votequorum_trackstart 3

"allows to" was updated to read "allows one to"

 - With a subject it's grammatically correct.

Change-Id: I9559e31c780e211b651744c6eaa056ce8d4c3db1
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 man/votequorum_trackstart.3.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/votequorum_trackstart.3.in b/man/votequorum_trackstart.3.in
index 34f3816..b440713 100644
--- a/man/votequorum_trackstart.3.in
+++ b/man/votequorum_trackstart.3.in
@@ -52,7 +52,7 @@ function that will execute the callback.
 .PP
 The
 .I context
-option allows to set a tracking context.
+option allows one to set a tracking context.
 .PP
 The
 .I flags
